brian2/stateupdaters/base.py

```diff
@@ -127,10 +138,16 @@ def apply_stateupdater(equations, variables, method):
             if the_method is None:
                 raise ValueError(('No stateupdater that is suitable for the '
                                   'given equations has been found.'))
-            logger.info('Using state updater: %r' % the_method)
+            if group_name is not None:
+                msg_text = ("No numerical integration method specified for group "
+                            "'{group_name}', choosing method '{method}'.")
+            else:
+                msg_text = ("No numerical integration method specified, "
+                            "choosing method '{method}'.")
+            logger.info(msg_text.format(group_name=group_name,
+                                        method=the_method), 'method_choice')
```

Comment:
I don't quite see the idea of the two messages? Is the point to see how long it's taking to select a method? I think it's too verbose for info level, and I'm not sure how useful it would be at debug level (but I have no objection to it there, it may be useful in some cases). I think more useful would be a message at info level saying that choosing the integration method took X seconds if X>some threshold. We can do both of course.